src/NServiceBus.Core/Diagnostics/Metrics/ReceiveDiagnosticsBehavior.cs

```diff
@@ -34,7 +25,7 @@ public async Task Invoke(IIncomingPhysicalMessageContext context, Func<IIncoming
                 new("nservicebus.type", messageTypes ?? ""),
             };
 
-            TotalFetched.Add(1, tags.ToArray());
+            MessagingMetricsFeature.TotalFetched.Add(1, tags.ToArray());
```

Comment:
We've been moving the counters in and out of the behavior, but this feels weird. Maybe the counters can be in one place instead, e.g. `NServiceBusCounters` with all the counters in one class and link to that from the behavior/feature?